core/domain/question_domain_test.py

```diff
@@ -121,7 +121,7 @@ def setUp(self):
         question_state_data = self._create_valid_question_data('ABC')
         self.question = question_domain.Question(
             'question_id', question_state_data,
-            feconf.CURRENT_STATE_SCHEMA_VERSION, 'en', 1)
+            feconf.CURRENT_STATE_SCHEMA_VERSION, 'en', 1, [])
```

Comment:
Since every question corresponds to at least one skill, should we be throwing an error if the list is empty?

Reply:
The reason I haven't done this is because the editors are already out right? Won't it throw errors for questions that already exist?

Reply:
I think I'm just gonna delete all questions on the test server before we push this, so let's just go ahead and do the right "ideal" thing now. There aren't any questions on the prod server yet fortunately.

Reply:
Apparently repeated fields cannot be set as required in ndb
<img width="785" alt="Screen Shot 2019-06-16 at 1 46 23 PM" src="https://user-images.githubusercontent.com/11780495/59569357-3221ee00-903d-11e9-9aeb-884892178e2d.png">

Reply:
Oh yes! Sorry you are right. This is probably more of a check for the validate() method in the domain object. (Though in that case I guess we can add it later? Probably want to file an issue to track.)